test/functional/specs/Privacy/C14414.js

```diff
@@ -29,19 +28,17 @@ test.meta({
   TEST_RUN: "Regression"
 });
 
-test("Test C14414: Requests are queued while consent changes are pending", async t => {
+test.only("Test C14414: Requests are queued while consent changes are pending", async t => {
   const alloy = createAlloyProxy();
   await alloy.configure(config);
   const setConsentResponse1 = await alloy.setConsentAsync(CONSENT_IN);
   const setConsentResponse2 = await alloy.setConsentAsync(CONSENT_OUT);
-  const logger = await createConsoleLogger();
   await alloy.sendEvent();
-  await logger.warn.expectMessageMatching(/user declined consent/);
-  await t
-    .expect(setConsentHook.haveRequestsBeenSequential())
-    .ok("Set-consent requests were not sequential");
 
   // make sure there are no errors returned from the setConsent requests
   await setConsentResponse1.result;
   await setConsentResponse2.result;
+
+  // make sure the event was not sent
+  await t.expect(networkLogger.edgeEndpointLogs.requests.length).eql(0);
```

Comment:
It seems like this will test that the responses came back in the right order (in and then out), but not necessarily that the requests went out sequentially in the right order, which seems important too. Thoughts?